server/src/routes/rootRouter.js

```diff
@@ -11,6 +11,4 @@ rootRouter.use("/api/v1/builds", buildsRouter)
 rootRouter.use("/api/v1/user", userDataRouter)
 rootRouter.use("/", clientRouter);
 
-//place your server-side routes here
-
-export default rootRouter;
+export default rootRouter;
```

Comment:
files should end with a single empty line
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--newline-at-end

Reply:
done

Reply:
nick said ok